src/useRouteQueryParam/formats/NullableBooleanRouteParam.ts

```diff
@@ -0,0 +1,27 @@
+import { LocationQueryValue } from 'vue-router'
+import { InvalidRouteParamValue } from '@/useRouteQueryParam/formats/InvalidRouteParamValue'
+import { RouteParam } from '@/useRouteQueryParam/formats/RouteParam'
+
+export class NullableBooleanRouteParam extends RouteParam<boolean | null> {
+
+  protected override parse(value: LocationQueryValue): boolean | null {
+    if (value === 'true') {
+      return true
+    }
+
+    if (value === 'false') {
+      return false
+    }
```

Comment:
any reasonable way to reuse the `BooleanRouteParam` class here?

Reply:
Not without refactoring something. `BooleanRouteParam` extends `RouteParam<boolean>` which is what sets the types for the `parse` and `format` methods. Extending `BooleanRouteParam` and returning `null` throws ts errors. 

Very open to ideas here. But didn't think it was worth putting to much effort into reusing a few lines of code.